packages/web-forms/env.d.ts

```diff
@@ -3,3 +3,6 @@
 /// <reference path="../../vendor-types/primevue/events.d.ts" />
 /// <reference path="../../vendor-types/primevue/dropdown.d.ts" />
 /// <reference path="../../vendor-types/primevue/multiselect.d.ts" />
+/// <reference path="../../vendor-types/@vue/input-html-attributes.d.ts" />
+
+declare const __WEB_FORMS_VERSION__: string;
```

Comment:
Fine for now, but for future reference, I have found it better to do something like:

```ts
declare global {
	// eslint-disable-next-line no-var
	declare var __WEB_FORMS_VERSION__: string | undefined;
}
```

... and then:

1. Reference the value as a property on `globalThis`
2. Do an explicit non-nullish check

All other variables using this pattern are using a regular `CONST_CASE` naming convention (without prefix/suffix). Let's not change this now, I think it'll be good to consider whether there's a better naming convention across the board.